mtp_cashbook/templates/core/index.html

```diff
@@ -1,15 +1,22 @@
 {% extends "base.html" %}
 
-{% block title %}Yo{% endblock title %}
-
 {% block content %}
-<div class="row">
-  <div class="small-12 columns moj-wrapper">
-    <h1>
-      Yo MoJ222!&nbsp;
-      <a href="//github.com/mixxorz/generator-moj-django/"><i class="fa fa-github"></i></a>
-    </h1>
-    <p>Welcome to your MoJ Django app.</p>
-  </div>
-</div>
+  <h1>Money sent to prisoners</h1>
+
+  <ul class="SummaryBoxes">
+    <li class="SummaryBoxes--fresh">
```

Comment:
`.SummaryBoxes--fresh` denotes a variation (hence `--`) of a `li` in the `.SummaryBoxes` module. It's a naming convention base on [SUIT CSS](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md), which is similar but not as dogmatic as "BEM". I'm using this as a prefix to any sub elements which require a class.

The CamelCase is a convention I am using which provides an easy way to identify the root element of HTML modules. This will become clearer as I create more modules. I will check this with the browsers we need to support.